akka-remote/src/test/scala/akka/remote/transport/AkkaProtocolStressTest.scala

```diff
@@ -44,7 +44,8 @@ object AkkaProtocolStressTest {
   object ResendFinal
 
   class SequenceVerifier(remote: ActorRef, controller: ActorRef) extends Actor {
-    import context.dispatcher
+    import scala.concurrent.ExecutionContext
+    private implicit val ec: ExecutionContext = context.dispatcher
```

Comment:
order of modifiers please like this (everywhere): `implicit private val` so it's very visible that it's an implicit

Reply:
`private implicit val` seems to be more widespread than `implicit private val` - https://github.com/scala/docs.scala-lang/issues/1075#issuecomment-395631176
Commonly used construct would take less time to percept when skimming through the code.